_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,116 @@
+
+- It will take you 3-5 months to release your changes to production.
+- You will deal with more than 13 stakeholders, both internal and external, with distinct functional roles as well as different intake and handoff processes.
+- You will go through more than 48 manual handoffs which consist of sending emails and opening tickets.
+- If releasing two weeks of work means it takes 3-5 months to release we have decreased our productivity by a factor of _3 (months) x 4 (weeks/month) / 2 ( weeks/unit of work)_ = **6** units of work to _5 (months) * 4 (weeks/month) / 2 (weeks/unit of work)_ = **10** units of work.
+
+Not only have we decreased our productivity by a factor of 6-10 but we have also created a large lead time which reduces our ability to respond to change. [Mark Schwartz](
+https://itrevolution.com/faculty/mark-schwartz/), a serial CIO, including that of US Homeland Security, who is now an Enterprise Strategist at Amazon Web Services, describes a perfect analogy of why this is detrimental in his book War and Peace and IT. Mark [draws an analogy](https://www.youtube.com/watch?v=2BM0xYfcexY) to The Battle of Borodino and the software delivery approach a lot of large organizations take.  In the battle, Napoleon is stationed a mile away from the battlefield and issues commands to his army through messengers. However, by the time the messengers deliver news it is already outdated and thus the commands that Napoleon issues do not reflect the current needs of his army. A famous example of this is when the bridge at Borodino is taken by Napoleon's army. A messenger is sent and delivers this news to Napoleon. So, Napoleon instructs that the army should cross the bridge. However, what Napoleon doesn't realize is by the time the messenger delivered the message, the bridge had not only been retaken but burned down such that it was no longer able to be crossed. This lead time of decision making is influenced by our current process which impacts our ability to deploy quickly to production. By the time a change is actually delivered, requirements have already changed and we are constantly playing a game of catch up. Manual processes, tests, checks and hand-offs also mean a larger potential of error and a greater risk of non-compliance for items such as Accessibility and IT Security. There is simply too much to be able to check manually and this leads to overburdening our operational staff as well as a large amount of waiting and toil for developers to be able to incorporate change. The result is that we are focused on putting out fires rather than looking at areas of improvement and addressing underlying issues.
```

Comment:
Comma between 'news' and 'it'?  in 'by the time the messengers deliver news it is already...'?